core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.spec.ts

```diff
@@ -422,19 +414,51 @@ describe('Contributor dashboard Admin page', () => {
       expect(component.selectedLanguage.id).toBe(nonDefaultLanguage.id);
     }));
 
-    it('should select last activity from dropdown', fakeAsync(() => {
+    it(
+      'should initially filters users by whether their last activity occurred ' +
```

Comment:
should...filter, not should...filters. Also please check other test names

Reply:
done